pkg/kn/commands/service/service_create_test.go

```diff
@@ -467,3 +467,30 @@ func TestServiceCreateEnvForce(t *testing.T) {
 		t.Fatalf("wrong output: %s", output)
 	}
 }
+
+func TestServiceCreateLabel(t *testing.T) {
+	action, created, _, err := fakeServiceCreate([]string{
+		"service", "create", "foo", "--image", "gcr.io/foo/bar:baz", "-l", "a=mouse", "--label", "b=cookie", "--async"}, false, false)
```

Comment:
just wondering, does the option parsing support: `--label=foo=bar` ? If so, we should probably test for that variant too.

Reply:
I'm not sure what you mean, there's already a `--label b=cookie`. Unless you mean the difference between `--label=foo=bar` and `-label foo=bar`?

Reply:
Yes I mean —label=foo=bar . Some options parsers support the merging of the option name with the value all in one cmd line arg separated but an =  So if ours does we should probably test it.  Especially in the case f removing a label.

Reply:
It looks like cobra does support that. I added a test case to `service_update_test.go` specifically for removing a label using that format. However, it doesn't seem that any other flag test uses that formatting.